client/go/cmd/testdata/tests/expected-suite.out

```diff
@@ -1,8 +1,11 @@
-Running my test: .... OK
-Running wrong-bool-value.json: Failed step 1:
-Unexpected value at /root/coverage/full: true
+My test: .... OK
+wrong-bool-value.json: failed
+Step 1:
```

Comment:
`In step 1`  may be better?

Reply:
Hmm, yes, but it doesn't really match `<step name>"` ...